cli/azd/pkg/ext/models.go

```diff
@@ -67,6 +70,9 @@ type HookConfig struct {
 	Windows *HookConfig `yaml:"windows,omitempty"`
 	// When running on linux/macos use this override config
 	Posix *HookConfig `yaml:"posix,omitempty"`
+	// Hides executing messages printed to the console
+	// This is a temp workaround until we have a better way to handle this
+	Quiet bool
```

Comment:
Can you add `yaml:"-"` here so that we don't support setting `Quiet` in `azure.yaml`?  Also, perhaps file an issue and tag it here so we can remember to revisit it?